src/app/content/studyGuides/components/ColorKey.spec.tsx

```diff
@@ -50,19 +48,47 @@ describe('Study Guides button and PopUp', () => {
       </Services.Provider>
     </Provider>);
 
-    /*open and close color key*/
+    /*open and close color key from button*/
     const button = component.root.findByType(ColorKeyButtonWrapper);
     renderer.act(() => {
       button.props.onClick();
     });
     expect(spyTrack).toHaveBeenCalledWith({pathname: '/'}, true);
 
     renderer.act(() => {
-      ReactTestUtils.Simulate.click(component);
-      button.props.onClick();
+      button.props.onClick(); /* closing from Color key button*/
     });
+
     /*mock close call to GA*/
     services.analytics.openCloseColorKey.track({pathname: '/'}, false);
-    expect(spyClickOutside).toHaveBeenCalledWith({current: null}, true, expect.any(Function));
+    expect(() => component.root.findByType(ColorKeyDescription)).toThrow();
```

Comment:
I believe that if the event should no longer be dispatched on `close` then: 
 - it's better to assert that it actually doesn't get sent after closing
 - rename it so it's name indicates that it's only for opening  sg

Reply:
It will be implemented eventually, that's why I didn't modify much.

Reply:
Line 63 is just for code coverage on sending false, since we don't want the GA event right now but will need later.